trading/views.py

```diff
@@ -2,15 +2,160 @@
 Views for trading
 """
 from django.shortcuts import render
+from django.http import HttpResponse
+from django.contrib.auth.decorators import login_required
+from django.core.exceptions import ObjectDoesNotExist
 from stocks.models import InvestmentBucket
+from trading.models import TradingAccount, TradeBucket
+from authentication.views import get_balance_as_dic
+from collections import defaultdict
 
 
-def test(request):
+@login_required
+def create_account(request):
```

Comment:
Let's just make a default trading account when the profile is created

Reply:
and not allow further accounts at all, or in addition to?

Reply:
For now, no further creation. Once we have solidified the model, we can expand into multiple